static/analytics.ts

```diff
@@ -22,47 +22,13 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import {options} from './options.js';
-
 class GAProxy {
-    private hasBeenEnabled = false;
     private isEnabled = false;
     private _proxy: (...args) => void = () => {};
 
     initialise() {
-        if (!this.isEnabled && options.statusTrackingEnabled && options.googleAnalyticsEnabled) {
-            // Check if this is a re-enable, as the script is already there in this case
-            if (!this.hasBeenEnabled) {
-                (function (i, s, o, g, r, a?: any, m?: any) {
-                    i.GoogleAnalyticsObject = r;
-                    i[r] =
-                        i[r] ||
-                        function () {
-                            // We could push the unexpanded args, but better might have some unintended side-effects
-                            // eslint-disable-next-line prefer-rest-params
-                            (i[r].q = i[r].q || []).push(arguments);
-                        };
-                    i[r].l = Date.now();
-                    a = s.createElement(o);
-                    m = s.getElementsByTagName(o)[0];
-                    a.async = 1;
-                    a.src = g;
-                    m.parentNode.insertBefore(a, m);
-                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
-                window.ga('set', 'anonymizeIp', true);
-                window.ga('create', options.googleAnalyticsAccount, {
-                    cookieDomain: 'auto',
-                    cookieFlags: 'SameSite=None; Secure',
-                });
-                window.ga('send', 'pageview');
-            }
-            this._proxy = (...args) => window.ga.apply(window.ga, args);
-            this.isEnabled = true;
-            this.hasBeenEnabled = true;
-        } else {
-            this.isEnabled = false;
-            this._proxy = () => {};
-        }
+        this.isEnabled = true;
```

Comment:
This "`isEnabled = true` seems dodgy from a person-idly-reading-the-code point of view. Can we not make this `false` - what breaks if we do?

Reply:
Nothing will break because the enabled status isn't read anywhere else in the code, toggle is always called with either true or false. It's not really an expected pattern to not set it to true, though. Perhaps it's just better to fully remove all implementation.